Docs/source/install/hpc/lumi.rst

```diff
@@ -13,12 +14,12 @@ If you are new to this system, **please see the following resources**:
 
 * `Lumi user guide <https://docs.lumi-supercomputer.eu>`_
 * Batch system: `Slurm <https://docs.lumi-supercomputer.eu/runjobs/scheduled-jobs/slurm-quickstart/>`_
-* `Jupyter service ? <TODO>`__
-* Production directories:
+* `Data analytics and visualization <https://docs.lumi-supercomputer.eu/hardware/lumid/>`_
+* `Production directories <https://docs.lumi-supercomputer.eu/storage/>``:
```

Comment:
Double _ to avoid reference name clashes and a fix:
```suggestion
* `Data analytics and visualization <https://docs.lumi-supercomputer.eu/hardware/lumid/>`__
* `Production directories <https://docs.lumi-supercomputer.eu/storage/>`__:
```

Reply:
Thanks!